org-formation/organization.yaml

```diff
@@ -627,6 +627,19 @@ Organization:
         budget-alarm-threshold: 10000
         budget-alarm-threshold-email-recipient: aws-dpe-prod@sagebase.org
 
+  ScceDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-scce-dev
+      RootEmail: aws-scce-dev@sagebase.org
+      Alias: org-sagebase-scce-dev
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        AccountOwner: thomas.yu@sagebase.org
+        CostCenter: NO PROGRAM / 000000
```

Comment:
@thomasyu888 should this SCCE dev account have a specific `CostCenter`?

Reply:
No - this is just for SCCE team to explore resources.